src/models/ComponentLink.js

```diff
@@ -8,38 +8,36 @@ class ComponentLink extends FileInformation {
   /**
    * Default constructor.
    *
-   * @param {Object} [props.from={}] - Where the link is created.
-   * @param {String} [props.from.id] - ID of the component.
-   * @param {Object} [props.to={}] - Target of the link.
-   * @param {String} [props.to.id] - ID of the component.
-   * @param {String} [props.type] - Representation of the link.
+   * @param {String} [props.sourceRef] - Id of component can be the source in a link.
+   * @param {String} [props.targetRef] - Id of component can be the target of the link.
```

Comment:
why sourceRef and targetRef here and not everywhere we found it?

Reply:
done